docker-entrypoint.sh

```diff
@@ -2,21 +2,22 @@
 
 # if command is sshd, set it up correctly
 if [ "${1}" = 'sshd' ]; then
-  set -- /usr/sbin/sshd -D
-
-  # Setup SSH HostKeys if needed
-  for algorithm in rsa dsa ecdsa ed25519
-  do
-    keyfile=/etc/ssh/keys/ssh_host_${algorithm}_key
-    [ -f $keyfile ] || ssh-keygen -q -N '' -f $keyfile -t $algorithm
-    grep -q "HostKey $keyfile" /etc/ssh/sshd_config || echo "HostKey $keyfile" >> /etc/ssh/sshd_config
-  done
-  # Disable unwanted authentications
-  perl -i -pe 's/^#?((?!Kerberos|GSSAPI)\w*Authentication)\s.*/\1 no/; s/^(PubkeyAuthentication) no/\1 yes/' /etc/ssh/sshd_config
-  # Disable sftp subsystem
-  perl -i -pe 's/^(Subsystem\ssftp\s)/#\1/' /etc/ssh/sshd_config
+  shift 1
+  set -- /usr/sbin/sshd -D -e "$@"
```

Comment:
Could you please make enabling "-e" an option (i.e. presence of SSH_DEBUG env var)?

Reply:
Hello,

I could do that but is it really a good idea to do so ?

I mean "-e" is not a debug mode or anything. It just prints out on stdout/stderr and it's a common good practice to do so in docker containers.

I need to test It, but I think if one really wants to restore the previous behavior, he can do it by chaining a "-E" in his command. Sshd should consider the last option and not the "-e".